app/views/tracks/show/unjoined.html.haml

```diff
@@ -38,37 +38,15 @@
               Card.Card(1999); // => "Happy 1999 from me!"
               Card.Card(sender: "mom"); // => "Happy 2020 from mom!"
```

Comment:
@ErikSchierboom You could do this too while you're in this headspace.

Reply:
Sure. We'll store this in the `config.json` I guess? Would `snippet` or `code_snippet` be a good name?

Reply:
And the text to the left of that code snippet, want me to also store and load that from the `config.json`? It is in the "About C#" section.

Reply:
Atm, it's a file in the v2 tracks directory. `SNIPPET.txt` IIRC. We should probably leave it as it is, and read that file?

Reply:
> And the text to the left of that code snippet, want me to also store and load that from the config.json? It is in the "About C#" section.

Yes pls. Again, probably read it from the same place as it's stored in v2.

Reply:
Will do.